src/components/dropdown/DropdownOption.tsx

```diff
@@ -22,6 +22,8 @@ export interface DropdownItem {
   filterByProp?: {
     name: string;
     value: any;
+    secondaryName: string;
+    secondaryValue: any;
```

Comment:
is it really any? Probably **string** is the best choice here/

Reply:
@Ihor-Lavrov yes, agreed. For now only `value` might be not a string, but an array also. Will update

Reply:
@Ihor-Lavrov updated